Modules/System/Time Zone Management/Permissions/TimeZoneMgtObjects.PermissionSet.al

```diff
@@ -0,0 +1,10 @@
+permissionset 8720 TimeZoneMgtObjects
```

Comment:
I think that the permission set should be included in objects PermissionSet 219 "System Application - Objects". Otherwise this permission set is not accessible nor assignable.

Reply:
That would make sense in and of itself. @JesperSchulz, you indicated below that this should have its own public permission set.

> This permission set should probably be called "Time Zone Management - Objects", just like the caption. This one should be internal. You could then create a new permission set, which isn't internal and has the name "Time Zone Management - Execute". For now, that permission set would only contain the objects one...

Waiting for feedback on this one.

Reply:
After thinking this over, I agree with @miljance that adding it to the "System Application - Objects" would suffice. A separate assignable time zone permission seems impractical.